airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -240,8 +240,12 @@ protected boolean dbtFromDefinition() {
         .orElse(false);
   }
 
+  protected String getDestinationDefinitionKey() {
+    return getImageNameWithoutTag();
```

Comment:
Is there a reason the method `getImageNameWithoutTag` isn't re-used as the Override-able method?

https://github.com/airbytehq/airbyte/blob/0974a05c55bfd235319fc76f1a86f0e080a5fa46/airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java#L129-L131

Reply:
AFAICT, `getImageNameWithoutTag` is used in other methods so I didn't want to increase my changes blast radius. Also, its not _technicially_ the image name w/o tag. Its the key to use when parsing a definition yaml file to pull in options.

Reply:
The value produced for these methods are different, for `ClickhouseDestinationStrictEncryptAcceptanceTest` the values are 
```java
getImageNameWithoutTag() -> "airbyte/destination-clickhouse-strict-encrypt";
getDestinationDefinitionKey() -> "airbyte/destination-clickhouse"
```
My gut is saying that since this work is really to unblock other work, I am going to keep the scope as tight as possible and leave a refactor for another day.